File: 3st/Hyeonseo/programmer/배열의 길이에 따라 다른 연산하기.java
class Solution {
    public int[] solution(int[] arr, int n) {
        for(int i = 0; i < arr.length;i++) {
			if(arr.length % 2 == 0) {
                if(i % 2 == 1) {
				    arr[i] += n;
                }
			}
            else {
                if(i % 2 ==0) {
                    arr[i] += n;
                }
            }
		}
        return arr;
    }
}

File: 2st/Yeonwoo/배열의 길이에 따라 다른 연산하기.py
def solution(arr, n):
    answer = []
    arr_len = len(arr)
    if arr_len%2==1:
        for i, num in enumerate(arr):
            if i%2==0:
                answer.append(num+n)
            else:
                answer.append(num)
    else:
        for i, num in enumerate(arr):
            if i%2==1:
                answer.append(num+n)
            else:
                answer.append(num)
    return answer